src/js/common/schemaform/definitions/address.js

```diff
@@ -66,18 +64,23 @@ export function uiSchema(label = 'Address', useStreet3 = false) {
       const isRequired = addressSchema.required.length > 0;
 
       let stateList;
+      let labelList;
       if (country === 'USA') {
         stateList = usaStates;
+        labelList = usaLabels;
```

Comment:
I dislike switch statements in JS. They're easy to mess up, especially if you're not returning from each branch.